theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -5,28 +5,14 @@ set -xe
 echo "--Apt Hook Setup"
 
 pbuilder_hooksdir=/etc/pbuilder/${os}64/hooks
+local_hooksdir=$(PWD)/hooks-${os}-${version}-${BUILD_ID}
 
-# Cleanup any old hooks
-sudo rm -f ${pbuilder_hooksdir}/F60addforemanrepo
+cp --recursive --preserve=mode ${pbuilder_hooksdir} ${local_hooksdir}
```

Comment:
`pbuilder` doesn't accept multiple `--hookdir`s (well, it does, silently only using the last one), so we need to compose a hookdir with all the static hooks we deploy via Puppet *plus* the one "dynamic" we generate in here.

Reply:
Perhaps that's a good comment in the script itself?